src/index.js

```diff
@@ -0,0 +1,27 @@
+const GitBook   = require('gitbook-core');
+const { React } = GitBook;
+
+const Youtube = React.createClass({
+    propTypes: {
+        youtubeUrl: React.PropTypes.string.isRequired
+    },
+
+    render() {
+        const { youtubeUrl } = this.props;
+
+        return (
+            <div style="position: relative;padding-bottom: 56.25%;padding-top: 25px;height: 0;">
```

Comment:
We should use a CSS file for this (with a class `YoutubeBlock` or something), so that themes/plugins can change it if needed.